tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3598,6 +3611,13 @@ tsl::StatusOr<cudnn_frontend::Tensor> CreateCudnnTensor(
   RETURN_MSG_IF_CUDNN_ERROR(tensor);
   return tensor;
 }
+
+tsl::StatusOr<cudnn_frontend::Tensor> CreateCudnnTensor(
+    cudnn_frontend::Tensor original, int64_t uid, dnn::DataType dtype,
+    bool is_virtual = false) {
+  return tsl::errors : Internal("Not implemented.");
```

Comment:
I think this might be supported but the functionality is only used when we require cuDNN version of at least 8.9 and I can't easily test it. One option would be to give the clone overload of CreateCudnnTensor its own 8.9 version guard instead of sharing it with the existing overload and remove this.

Reply:
I'm fine either keeping it as-is or adding an 8.9 version check inside thedefinition of CreateCudnnTensor.

Either way, check CUDNN_VERSION in cudnn_fused_conv_rewriter.cc though.